tests/virt_autotest/virt_utils.pm

```diff
@@ -655,7 +655,12 @@ sub collect_host_and_guest_logs {
 sub cleanup_host_and_guest_logs {
     my ($extra_logs_to_cleanup) = @_;
 
-    script_run("source /usr/share/qa/qa_test_virtualization/shared/standalone") if get_var('VIRT_AUTOTEST');
+    #Clean dhcpd and named services up explicity
+    if (get_var('VIRT_AUTOTEST')) {
+        script_run("brctl addbr br123;brctl setfd br123 0;ip addr add 192.168.123.1/24 dev br123;ip link set br123 up;ifconfig br123 up 192.168.123.1 netmask 255.255.255.0");
```

Comment:
ifconfig is replaced with ‘ip link set’ command since sle15. we'd better not use it anymore in our test.

Reply:
Sles before 15 needs this. Sles15 can just report no such command.

Reply:
ifconfig command was built as legacy package on sles system now, also, will be removed form sles support package list, imho.
Would be better to use "ip" command, instead of using 'ifconfig' command on sles system (12+).

Reply:
But it still needs on 11sp4

Reply:
no, I had confirm ip command on my sles11sp4 LTSS vm host,  there was existed ip command from 
iproute2 package now:
```
Repository: virtupdate.sles-11-sp4-64
Name: iproute2
Version: 3.0-5.3.2
Arch: x86_64
Vendor: SUSE LINUX Products GmbH, Nuernberg, Germany
Support Level: Level 3
Installed: Yes
Status: up-to-date
Installed Size: 1.7 MiB
Summary: Advanced Routing
Description: 
This package provides the tools ip, tc, and rtmon needed to use the new
and advanced routing options of the Linux kernel. The SUSE Linux
distribution has used this package for network setup since SuSE Linux 8.0.
```

Reply:
OK. I will check later then update if necessary.

Reply:
Although I think this can be kept and does no harm, I removed this 'ifconfig' thing after thinking twice. Done.